e2e/cypress/support/api/team.js

```diff
@@ -76,15 +60,23 @@ Cypress.Commands.add('apiGetTeamByName', (name) => {
     });
 });
 
-/**
- * Gets a list of all of the teams on the server
- * This API assume that the user is logged in as sysadmin
- */
-Cypress.Commands.add('apiGetAllTeams', () => {
+Cypress.Commands.add('apiGetAllTeams', ({page = 0, perPage = 60, includeTotalCount = false} = {}) => {
     return cy.request({
         headers: {'X-Requested-With': 'XMLHttpRequest'},
-        url: 'api/v4/teams',
+        url: `api/v4/teams?page=${page}&per_page=${perPage}&include_total_count=${includeTotalCount}`,
         method: 'GET',
+    }).then((response) => {
+        expect(response.status).to.equal(200);
+        let teams;
+        let totalCount;
+
+        if (includeTotalCount) {
+            teams = response.body.teams;
+            totalCount = response.body.total_count;
+        } else {
+            teams = response.body;
+        }
+        cy.wrap({teams, totalCount});
```

Comment:
I did not see any usage of includeTotalCount parameter in any of the tests. Is this for the future?

Reply:
I understand and I'm a fan of not adding anything if not to be used. My primary goal is to match this custom API command based on corresponding [API reference](https://api.mattermost.com/#tag/teams/paths/~1teams/get). Though no usage currently, I can see that this will help other discover that there's an option like this whenever they need total count of teams. I'm fine to remove if you recommend to do so.

Reply:
Yes, I think this would just add complexity to the test code. I am not sure of a usecase where this is essential to be validated from the test. It's required in UI, so that pagination can work, but from the test perspective, I feel its not required and will not be used.

Reply:
Done, PTAL again.